### ISSUE TITLE
The problem about the loss function of actor network

### ISSUE THREAD

**Issue by speedhawk** [aidudezzz/deepbots-tutorials#38]
Hi! Could you please tell me where can I find the method named 'torch_min' and the corresponding folder "torch._C._VariableFunctions"? 
Here is your code:

`action_loss = -torch_min(surr1, surr2).mean()`

After reading some sources, I knew that this function only appears in PyTorch original codes implemented by C\C++. But I still not found this method even I have open the Pytorch repositories. Without the exact perceptive of this method, I cannot subsequently understand the sense of the loss function of actor network in your code. Many thanks!!

**Comment by tsampazk** [comment]
Hey there! `torch_min` is actually [torch.min](https://pytorch.org/docs/stable/generated/torch.min.html) as you can see in the import [here](https://github.com/aidudezzz/deepbots-tutorials/blob/787c4c1127d042af8b300f166f806b0f706f1fd2/robotSupervisorSchemeTutorial/full_project/controllers/robot_supervisor_controller/PPO_agent.py#L8).
